<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewDatabaseWizardPageGeneral</class>
 <widget class="QWizardPage" name="NewDatabaseWizardPageGeneral">
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>General Database Information</string>
  </property>
  <property name="subTitle">
   <string>Please fill in the name, and optional description and default username for your new database:</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="databaseNameLabel">
     <property name="text">
      <string>Database Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="databaseName"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="databaseDescriptionLabel">
     <property name="text">
      <string>Description:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="databaseDescription"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="databaseDefaultUsernameLabel">
     <property name="text">
      <string>Default Username:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="databaseDefaultUsername"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
